doc/api/buffer.md

```diff
@@ -869,6 +869,10 @@ The index operator `[index]` can be used to get and set the octet at position
 `index` in `buf`. The values refer to individual bytes, so the legal value
 range is between `0x00` and `0xFF` (hex) or `0` and `255` (decimal).
 
+Since this operator is inherited from `Uint8Array`, its behavior on
+out-of-bounds access is exactly the same - i.e. getting returns `undefined`,
+setting does nothing.
+
```

Comment:
> Nikolai uses i.e. correctly here as far as I can tell: "id est", not "exempli gratia."

Good to know, but the fact that I couldn't tell kind of proves my point, I think.

I'm concerned about the reader either thinking `i.e.` means "for example" or else knowing better but not being certain the docs author does. That second one is my experience pretty much every time I see `i.e.` on the web, especially in open source documentation. I have to stop and figure out if the author really meant `i.e.` or if they meant "for example". And sometimes it's hard to tell. Here's an example from our own docs (`addons.md`):

> Because the exact path to the compiled Addon binary can vary depending on how
it is compiled (i.e. sometimes it may be in `./build/Debug/`), Addons can use
the [bindings][] package to load the compiled module.

Reading that, I'm not sure if `./build/Debug/` is the *only* alternative to the implied canonical `./build/Release/` or if it's possible that there are other locations too. Even if `i.e.` is being used correctly in this case, it results in confusion and ambiguity (for me, at least) because I'm not sure that it's being used correctly.

For that reason, I always suggest eliminating `i.e.` (and `e.g.`) and just writing out what you mean to say. With `i.e.`, a good replacement depends on context. It may be nothing at all (just remove `i.e.`) or "that is" or "that is to say" or "in other words" or even something like "therefore".

That said, I'm OK with `i.e.` if people think it's the better option. I *prefer* we don't use it, and I've explained above, but reasonable people can certainly have a different opinion than mine on this. :-D